homework/task_6/Valentyn.Rudenko/Task6_3.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+
+public class Task6_3 {
+    public static void main(String[] args) {
+        int n, arr[];
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Please input length of massive: ");
+        n = sc.nextInt();
+        arr = new int[n];
+        for (int i = 0; i < arr.length; i++)
+            arr[i] = i + 1;  //(int) (Math.random() * n);
+      //  for (int i : arr);
```

Comment:
You don't need to left comments in the code. They are very misleading. Any previous changes can be viewed in git